lib/app/addons/ac/output-adapter.common.js

```
@@ -77,13 +78,14 @@ YUI.add('mojito-output-adapter-addon', function(Y, NAME) {
         meta.http.code = meta.http.code || 200;
         meta.http.headers = meta.http.headers || {};
         meta.view = meta.view || {};
+        appConfig = (configAddon.getAppConfig && configAddon.getAppConfig()) || {};
```

I prefer to keep it cleaner.  If `store.config` is missing that's quite exceptional, and we should fail early.